managed/services/management/node.go

```diff
@@ -136,10 +140,28 @@ func (s *NodeService) Register(ctx context.Context, req *managementpb.RegisterNo
 		return nil, e
 	}
 
-	apiKeyName := fmt.Sprintf("pmm-agent-%s-%d", req.NodeName, rand.Int63()) //nolint:gosec
-	_, res.Token, e = s.akp.CreateAdminAPIKey(ctx, apiKeyName)
-	if e != nil {
-		return nil, e
+	// get authorization from headers.
+	md, ok := metadata.FromIncomingContext(ctx)
+	if !ok {
+		msg := "Couldn't create Admin API Key: cannot get headers from metadata"
```

Comment:
we discussed with GAS team and made decission that register shouldn't fail in case it's not possible to create an API key. So pmm-agent will keep using user credentials instead of API key.